apps/rehabstod/tailwind.config.ts

```diff
@@ -17,6 +17,7 @@ export default {
     colors: {
       black: '#000',
       white: '#FFF',
+      orange: '#D17200',
```

Comment:
Looking at figma it looks like this color should be `#FF9517` same as "IDS-COLOR-GRAPHIC"